### PR TITLE
Update website for 2021-22

### PR DESCRIPTION
**Pull request by pettiboy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3436,17 +3438,17 @@ <h6><a href="https://www.codechef.com/users/mayor_9" target="_blank">Codechef: @
...
-          <li class="talk wow fadeInUp" id="bhuvnesh-solanki">
+          <li class="talk wow fadeInUp" id="jayesh-nirve">
             <h5 class="time"></h5>
             <div class="subject">
               <div class="description">
-                <img src="static/images/currentC/bhuvnesh_solanki.jpg" height="96" class="avatar" id="avatar" alt="">
-                <h5>Bhuvnesh Solanki</h5>
+                <img src="static/images/team-2021-22/Jayesh_2021-22.jpg" height="96" class="avatar" id="avatar" alt="">
+                <h5>Jayesh Nirve</h5>
                 <h6>Tech team</h6>
```
Comment (nis130) [inline index.html:3447]: Make this "t" uppercase

File: index.html [hunk 2]
```diff
@@ -4607,6 +4636,105 @@ <h4>Ms. Sanika Bagwe</h4>
...
+      <div class="council-7">
+        <div class="baraja-demo">
+          <p style="color: #214760;font-size:24px">2020-21</p>
+          <ul id="council-7" class="baraja-container">
+            <li>
+              <h4>Mrs. Grishma Sharma</h4>
+              <p>Faculty Advisor</p>
+            </li>
+            <li>
+              <h4>Mrs. Dipti Pawade</h4>
```
Comment (nis130) [inline index.html:4649]: Add Avani maam's name here

**Comment by nis130** [comment]
@pettiboy Thanks!

LGTM, we can merge now @neelansh15 @twilight-warlock

**Comment by neelansh15** [comment]
Sure

**Pull request merged**